lib/http-bridge/src/Cardano/Wallet/HttpBridge/Compatibility.hs

```diff
@@ -169,3 +179,17 @@ byronSlotLength = SlotLength $ secondsToDiffTime 20
 -- | Hard-coded max transaction size
 byronTxMaxSize :: Quantity "byte" Word16
 byronTxMaxSize = Quantity 8192
+
+byronBlockchainParameters
+    :: forall n. KnownNetwork n
+    => BlockchainParameters (HttpBridge n)
+byronBlockchainParameters = BlockchainParameters
+    { getGenesisBlock = block0
+    , getGenesisBlockDate = case networkVal @n of
+        Mainnet -> posixSecondsToUTCTime 1506203091
+        Testnet -> posixSecondsToUTCTime 1537941600
```

Comment:
Corrected from the other PR https://github.com/input-output-hk/cardano-wallet/pull/529#discussion_r304274305